assets/scss/6-components/full-page-loader/_full-page-loader.scss

```diff
@@ -0,0 +1,51 @@
+$loader-bar-height: 10px;
+
+/*
+  No demo for this component because, if we made one,
+  it would cover the entire page!
+
+  This component is for putting the entire width and
+  height of a page in a loading/pending state.
+
+  Inspired by:
+  https://codepen.io/brunjo/pen/XJmbNz
+*/
+.c-full-page-loader {
+  background-color: rgba(255,255,255,.95);
```

Comment:
totally not necessary but we could use `background-color: rgba($color-white-off, .95);` for readability. Most people know the rgb of white, but maybe some don't and would interpret a color var more easily.

Also, fun fact: you can run `yarn lint` to stylelint the scss. This particular file barks about some spacing stuff. It likes line breaks, but we can adjust that rule in `package.json` if we want.

Reply:
Good call on the color variable -- I used `$color-white-pure` though.

I did not know that linting existed, and I enjoy it! We should integrate into CI later.

Reply:
doh, yes I meant pure. Indeed on the CI task!